contents/docs/product-analytics/group-analytics.mdx

```diff
@@ -24,10 +24,110 @@ import JSSetGroupProperties from "../\_snippets/setting-group-properties.mdx"
+
+To link a user to a group, call the same code as you would when [creating a group](#how-to-create-groups).
+
+### Advanced (server-side only): Capturing group events without a user
+
+If you want to capture group events but don't want to associate them with a specific user, there are two alternatives:
+
+#### Method one (recommended): Use the group ID as the event's distinct ID
+
+When capturing events, instead of using the user's ID as the distinct ID, you can use group ID as the distinct ID. For example:
+
+<MultiLanguage>
+
+```node
+posthog.capture({
+    event: 'group_event_name',
+    distinctId: 'group_id_in_your_db',
+})
+```
+
+```python
+posthog.capture('group_id_in_your_db', 
+    'group_event_name', 
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "group_id_in_your_db",
+    Event: "group_event_name",
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'group_id_in_your_db',
+    'event' => 'group_event_name'
+));
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "group_event_name",
+    "distinct_id": "group_id_in_your_db",
+}' <ph_instance_address>/capture/
+```
+
+This has the advantage that insights such as "total users and "unique users" in [trends](/docs/product-analytics/trends) will work as expected for these groups events (note: although it will say "users", it will actually mean "groups")
+
+The drawback of this approach is that a new [person](/docs/data/persons) will be created for each group. While this won't affect your data integrity in any way, it may cause confusion since these groups are not actually people.
+
+</MultiLanguage>
+
+#### Method two: Use the same ID as the event's distinct ID
+
+Instead of using the group's ID as the distinct ID, you can create a *single string* that will be used to capture group events. This can be anything you want, as long as it's the same string for every group event:
+
+```node
+posthog.capture({
+    event: 'group_event_name',
+    distinctId: 'static_string_for_group_events',
+})
+```
+
+```python
+posthog.capture('static_string_for_group_events', 
+    'group_event_name', 
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+    DistinctId: "static_string_for_group_events",
+    Event: "group_event_name",
+})
+```
+
+```php
+PostHog::capture(array(
+    'distinctId' => 'static_string_for_group_events',
+    'event' => 'group_event_name'
+));
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "event": "group_event_name",
+    "distinct_id": "static_string_for_group_events",
+}' <ph_instance_address>/capture/
+```
+
+The drawback of this approach is that insights such as "total users and "unique users" in [trends](/docs/product-analytics/trends) will not work as expected for these group events (since it is same ID submitting these events).
```

Comment:
If you just use one ID for group analytics, total users and unique users will be more accurate for normal persons (1 more unique user for groups)

Reply:
When in posthog would you calculate or use "total users" or "unique users" for ALL normal persons? 

These are done usually at an event level. Im not aware of other use cases?

Reply:
There is an `All events` series.

Reply:
Ah, didnt realize we had this! Thanks for point it out